plugins/single_plugins/vswitch.cpp

```diff
@@ -137,7 +137,7 @@ class vswitch : public wf::plugin_interface_t
     {
         if (!x && !y)
```

Comment:
It would be better to move this check to https://github.com/WayfireWM/wayfire/pull/723/files#diff-7746de7cb4788c85484ac7396b90b98bR198 as a whole, it does not belong here (not sure why it was put it here in the first place). Otherwise LGTM.